test/config-luatest/reload_test.lua

```diff
@@ -0,0 +1,80 @@
+local t = require('luatest')
+local treegen = require('test.treegen')
+local server = require('test.luatest_helpers.server')
+
+local g = t.group()
+
+g.before_all(function(g)
+    treegen.init(g)
+end)
+
+g.after_all(function(g)
+    treegen.clean(g)
+end)
+
+g.after_each(function()
+    if g.server ~= nil then
+        g.server:stop()
+    end
+end)
+
+g.test_no_cluster_config_failure = function(g)
+    local dir = treegen.prepare_directory(g, {}, {})
+    local config = [[
+        credentials:
+          users:
+            guest:
+              roles:
+              - super
+        iproto:
+          listen: unix/:./{{ instance_name }}.iproto
+        groups:
+          group-001:
+            replicasets:
+              replicaset-001:
+                instances:
+                  instance-001: []
```

Comment:
Nit: Technically it is not quite correct. This field accepts a record, not an array. It works, because an empty array and an empty map are represented in the same way in Lua.

Reply:
Thanks, fixed